Anchorage.xcodeproj/project.pbxproj

```diff
@@ -434,7 +434,7 @@
 					};
 					CD6A0E661EB7CDEF00FFF5DB = {
 						CreatedOnToolsVersion = 8.3.2;
-						LastSwiftMigration = 0900;
+						LastSwiftMigration = 1020;
```

Comment:
I updated the check (by unchecking _Remind Me_), but didn't change the Swift version, so that the Xcode warning would go away.